roles/famedly_sync/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- name: "Create volume path"
+  file:
+    path: "{{ item }}"
+    state: "directory"
+    mode: "0700"
+  loop:
+    - "{{ famedly_sync_path }}"
+
+- name: "Ensure container image is present locally"
+  community.docker.docker_image:
+    name: "{{ famedly_sync_container_image_reference }}"
+    source: "pull"
+    state: "present"
+    force_source: "{{ famedly_sync_container_image_force_pull }}"
+
+- name: "Ensure container is present"
+  community.docker.docker_container:
+    name: famedly-sync
+    image: "{{ famedly_sync_container_image_reference }}"
+    state: "present"
+    volumes: "{{ famedly_sync_docker_volumes }}"
+    labels: "{{ famedly_sync_docker_labels | default(omit, True) }}"
+    networks: "{{ famedly_sync_docker_networks }}"
+
+- name: "Deploy config"
+  copy:
+    content: "{{ famedly_sync_config | to_nice_yaml }}"
```

Comment:
Could you give this var a sensible default or if that doesn't make sense assert that it is defined in the beginning and add it to the README?